.cirrus.yml

```diff
@@ -736,6 +736,33 @@ upgrade_test_task:
     always: *logs_artifacts
 
 
+image_build_task:
+    name: "Build multi-arch images and manifest lists"
+    alias: image_build
+    skip: *tags
+    # FIXME: Run nightly on 'main' branch
+    # only_if: $CIRRUS_CRON == 'multiarch'
+    only_if: $CIRRUS_CHANGE_TITLE =~ '.*CI:BUILD.*'
+    depends_on:
+        - ext_svc_check
+    timeout_in: 120m  # emulation is sssllllooooowwww
+    gce_instance:
+        image_project: libpod-218412
+        zone: "us-central1-a"
+        # More muscle required for parallel multi-arch build
+        type: "n2-standard-4"
+        disk: 200
+        # FIXME: Use "build-push-c${IMAGE_SUFFIX}"
```

Comment:
It is not clear what conditions will lead to this being fixable. It will be even less clear in the future. Could you please document why this can't be done now, and what will enable this being fixed?